### ISSUE TITLE
ebib-index buffer reflow when fields contain non ltr scripts

### ISSUE THREAD

**Issue by oatmealm** [joostkremers/ebib#172]
The `ebib-index` buffer reflows columns like in the example, i.e. the author column.
I've tried forcing `bidi-paragraph-direction` as `left-to-right`.... with no effect.
The key part is properly aligned, but the rest re-flows from right to left, author, year, title. 

![gnome-shell-screenshot-9EXDP0](https://user-images.githubusercontent.com/68159077/90764808-d9a14d00-e2e0-11ea-9810-56d9663c0d14.png)

**Comment by joostkremers** [comment]
Hmm... That's a use-case I never considered. Could you send me a few .bib entries with RTL text to help with testing?

The columns are created using Emacs' built-in functionality, so I'll have to figure out if they are flexible enough to handle combinations of RTL and LTR text.

**Comment by oatmealm** [comment]
Yes, here are couple of short onees:

<details><pre>
@Book{bashan1999,
  file =         {bashan1999.pdf},
  pages =        204,
  keywords =     {יהדות מרוקו},
  author =       {בשן, אליעזר},
  publisher =    {אוניברסיטת בר-אילן},
  year =         1999,
  title =        {היהודים במרוקו במאה ה-19 והמיסיון האנגליקני}
}

@Book{bashan2000,
  pages =        409,
  publisher =    {הקיבוץ המאוחד},
  series =       {ספריית "הילל בן-חיים"},
  author =       {בשן, אליעזר},
  year =         2000,
  title =        { יהדות מרוקו, עברה ותרבותה}
}
</pre></details>

**Comment by oatmealm** [comment]
Wanted to add that I'm seeing similar problem in `helm-bibtex`'s display of such mixed script lists....

![gnome-shell-screenshot-1BJHQ0](https://user-images.githubusercontent.com/68159077/92327864-f36ebe00-f05c-11ea-8570-ceb363d5e222.png)

**Comment by joostkremers** [comment]
I sent a message to `help-gnu-emacs` about this issue. Hopefully I get some useful responses, because I don't really have an idea on how to handle this better...

**Comment by joostkremers** [comment]
Turns out this has all been thought of and taken care of by the Unicode standard and the Emacs devs. I've pushed a commit that should fix this issue. In my Emacs setup, Hebrew text is displayed by a variable-width font (don't ask me why...) so the columns do not line up, but your screen shots show a fixed-width font, so hopefully it'll work fine for you.

Do let me know if you still run into problems.
